hawk/config/routes.rb

```diff
@@ -107,18 +107,19 @@
 
   resources :agents, only: [:show]
 
-  resources :explorers, only: [:index, :destroy] do
+  resources :reports, only: [:index, :destroy, :show] do
     collection do
       post :generate
       post :upload
+      get :running, defaults: { format: 'json' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.